services/pdf-ms/templateExample/ekb-löpande.json

```diff
@@ -0,0 +1,480 @@
+{
+  "numberOfpages": 5,
+  "defaultFontSize": 11,
+  "texts":
+  [
+  {
+      "page": 0,
```

Comment:
more user friendly to start with page nr 1...

Reply:
Well, this is for programmers, and we should know that indices starts at zero, no?

Reply:
Maybe change the property to pageIndex instead @jwinding, to avoid confusion (y)